src/components/ErrorMessage/ErrorMessage.tsx

```diff
@@ -0,0 +1,56 @@
+import { ErrorSpec } from '../../types/ErrorSpec';
+
+type Props = {
+  error: ErrorSpec | null;
+};
+
+export const ErrorMessage: React.FC<Props> = ({ error }) => {
+  const renderErrorMessage = () => {
```

Comment:
If you want to keep it as a separate function - it is better to create it outside the component.
Also, all error tags are the same, so you only need to select the text and wrap it once in the correct tags